src/database/datatools/schedulecompact.py

```
@@ -0,0 +1,127 @@
+    cursor.execute(sql, (schedule_compact_id,))
+
+    conn.commit()
+    conn.close()
+
+def list_schedule_compact(db_path: str) -> List[Tuple]:
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+
+    table_name = "schedule_compact"
+
+    sql = f"SELECT * FROM {table_name}"
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)